sdk/keyvault/generate.ts

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+const { execSync } = require("child_process");
```

Comment:
The end result is that this file should be deleted once all the workarounds are no longer needed

Reply:
Should we have this at the top level? Or should we have one per package (in case there are package-specific things which need to be done, generating from a specific package's spec, etc)?

Reply:
Yeah, I can move it inside the package folder. My hope (and so far that's been the case) is that the same set of customizations need to happen for all packages. I figured if they start diverging I can push them down into the individual package folder. But my real hope is that this script gets deleted by end of January as its a workaround, not a solution.

Reply:
Yeah if there are no differences between packages no problem with just keeping the script here. Whatever is least work for you.